tools/shared.py

```diff
@@ -746,6 +746,7 @@ def safe_copy(src, dst):
   if dst == os.devnull:
     return
   shutil.copyfile(src, dst)
+  shutil.copymode(src, dst)
```

Comment:
why not just `shutil.copy2`?

Reply:
I'm not sure they are identical.. will double check and use that if so.

Reply:
Actually it looks like what I want is `shutil.copy` which means data + permissions bits but not timestamps.  `copy2` also does timestamps and other metadata.